### ISSUE TITLE
Wrong loss criterion

### ISSUE THREAD

**Issue by Karansaluja** [mjc92/GetToThePoint#8]
You are using **softmax + NLLLoss**
You should either use **log_softmax + NLLLoss** or **softmax + CrossEntropyLoss**
https://discuss.pytorch.org/t/is-log-softmax-nllloss-crossentropyloss/9352
